post_introspect.yml

```diff
@@ -70,7 +70,7 @@
       vars:
         image_path: /home/stack/images
       include_tasks: tasks/overcloud_images.yaml
-      when: osp_release|int == 17
+      when: remote_image_path is defined and osp_release|int == 17
```

Comment:
same as above

Reply:
this was deprecated in the PR https://github.com/redhat-performance/jetpack/pull/487 as now we are using overcloud-full image in bios mode